.github/workflows/scripts/pr.js

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env node
+
+const request = require('./request');
+
+const IN_REVIEW_LABEL = '[zube]: Review';
+const IN_TEST_LABEL = '[zube]: To Test';
+const DONE_LABEL = '[zube]: Done';
+
+// The event object
+const event = require(process.env.GITHUB_EVENT_PATH);
+
+function getReferencedIssues(body) {
+    // https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
+    const regexp = /[Ff]ix(es|ed)?\s*#([0-9]*)|[Cc]lose(s|d)?\s*#([0-9]*)|[Rr]esolve(s|d)?\s*#([0-9]*)/g;
+    var v;
+    const issues = [];
+    do {
+        v = regexp.exec(body);
+        if (v) {
+            issues.push(parseInt(v[2], 10));
+        }
+    } while (v);
+    return issues;
+}
+
+function hasLabel(issue, label) {
+    const found = issue.labels.find(l =>l.name.toLowerCase() === label.toLowerCase());
+    return !!found;
+}
+
+function removeZubeLabels(labels) {
+    return labels.filter(l => l.name.indexOf('[zube]') === -1);
+}
+
+async function resetZubeLabels(issue, label) {
+    // Remove all zube labels
+    const cleanLabels = removeZubeLabels(issue.labels);
+    console.log(`    Current Labels: ${cleanLabels}`);
+    // Add the 'to test' label
+    cleanLabels.push(label);
+    console.log(`    New Labels    : ${cleanLabels}`);
+
+    // Update the labels
+    const labelsAPI = `${issue.url}/labels`;
+    request.put(labelsAPI, {labels: cleanLabels});
```

Comment:
This should have `await`. Alternatively I _think_ you could remove `async` and `return request.put...`